### DISCUSSION TITLE
libnss3.so not found in azure asp.net webapp

### DISCUSSION THREAD

**Discussion by rameshrajappan-next** [hardkoded/puppeteer-sharp#2263] [Q&A]
I am trying to use puppeteer sharp on my asp.net core api hosted in azure (as .net core web app).  But i am getting an error "Failed to launch browser! /home/site/wwwroot/.local-chromium/Linux-1069273/chrome-linux/chrome: error while loading shared libraries: libnss3.so: cannot open shared object file: No such file or directory" . Please help

**Comment by kblok** [comment]
I think a normal web app won't have everything you need to install a browser. You should use a docker image instead.
